README.md

```diff
@@ -18,166 +18,4 @@ CloudFormation). If you try to use such a Lambda with an API Gateway
 **HTTP API** (`AWS::ApiGatewayV2::Api` in CloudFormation), it will
 return 500s.
 
-## Overview of the repo
-
-There are three main subdirectories of interest:
-
-* `wai-handler-hal`: the library that glues `wai` and `hal` together;
-* `wai-handler-hal-example`: A simple `servant` API set up to run on
-  both `hal` and `warp`; and
-* `wai-handler-hal-cdk`: A small [AWS
-  CDK](https://docs.aws.amazon.com/cdk/latest/guide/home.html)
-  application that deploys an API backed by `wai-handler-hal-example`.
-
-## Developing
-
-Instead of serving your application with something like
-[`warp`](https://hackage.haskell.org/package/warp), you will set up an
-executable that serves your application with
-[`hal`](https://hackage.haskell.org/package/hal). The necessary glue
-code looks something like this:
-
-```haskell
-import AWS.Lambda.Runtime (mRuntime)
-import Network.Wai (Application)
-import qualified Network.Wai.Handler.Hal as WaiHandler
-
-app :: Application
-app = undefined -- From Servant or wherever else
-
-main :: IO ()
-main = mRuntime $ WaiHandler.run app
-```
-
-### Local testing
-
-Compiled Lambda functions can be awkward to test, especially if you're
-relying on an API Gateway integration to translate HTTP
-requests. Consider defining a second executable target that serves
-your `Application` using `warp`, which you can use for local
-testing.
-
-### Caveats
-
-* The Lambda is never told the port that the API Gateway is listening
-  on. Most APIs will listen on `443` (HTTPS), so that's what's in this
-  example.
-
-* The Lambda is never told the HTTP version that the client uses when
-  talking with the API Gateway. We assume HTTP 1.1.
```

Comment:
This is all still true, right? Should we keep it?

Reply:
I think the Caveats sections probably belongs in `wai-handler-example` instead, but otherwise yes

Reply:
Never mind, it's just referencing the example code above

Reply:
The caveats section is actually about limitations of the library itself, so if that's unclear I'd love to see it reworded.